pkg/reconcile/reconcilers.go

```diff
@@ -420,7 +438,22 @@ func (r *defaultReconciler) buildNewDeployment(req ReaperRequest) (*appsv1.Deplo
 		}
 
 		if usernameEnvVar, passwordEnvVar, err := r.secretsManager.GetJmxAuthCredentials(secret); err == nil {
-			addJmxAuthEnvVars(deployment, usernameEnvVar, passwordEnvVar)
+			addAuthEnvVars(deployment, usernameEnvVar, passwordEnvVar)
+		} else {
+			req.Logger.Error(err, "failed to get JMX credentials", "deployment", key)
```

Comment:
Can you fix this logging statement like the previous ones?

Reply:
What's wrong with this one? It's the original one.

Reply:
I assume you meant the different key instead of targeting the original deployment like the old message